frontend/src/metabase/query_builder/analytics.js

```diff
@@ -5,7 +5,7 @@ export const trackNewQuestionSaved = (
   createdQuestion,
   isBasedOnExistingQuestion,
 ) => {
-  trackSchemaEvent("question", "1-0-1", {
+  trackSchemaEvent("question", "1-1-1", {
```

Comment:
Followed the same pattern that the `settings` snowplow events have where the schema seems to match the latest version - let me know if this isn't right

Reply:
I think it's okay to update the version where possible, but if we stick to [the semantic versioning](https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa?pvs=4#fa3b889daa6949698cc61030f0fcbf2b), `1-1-1` would be incompatible with the previous event since we bump the minor version. It's safe to bump it to `1-0-2` in this case.